index.js

```diff
@@ -45,12 +45,20 @@ class Store {
 
     const opts = { debounce: 0, indent: 2, home: os.homedir(), name, ...options };
 
-    this.name = opts.name || (opts.path ? utils.stem(opts.path) : 'data-store')
+    this.name = opts.name || (opts.path ? utils.stem(opts.path) : 'data-store');
     this.path = opts.path || path.join(opts.home, `${this.name}.json`);
     this.indent = opts.indent;
     this.debounce = opts.debounce;
     this.defaults = defaults || opts.default;
     this.timeouts = {};
+
+    // Allow override of read and write methods
+    if (typeof options.readParseFile == "function") {
+      this.readParseFile = options.readParseFile;
+    }
+    if (typeof options.writeFile == "function") {
```

Comment:
Please use single quotes and strict equals:

```suggestion
    if (typeof options.writeFile === 'function') {
```